CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.2.1 - 2017-12-06
+
+### Fixes
+
+- Fix handling of temporary read/write errors
+- Don't send initial Mixpanel events before "Anonymous Tracking" settings are loaded.
+- Fix verification step reading from the cache
```

Comment:
Huh? According to https://github.com/resin-io/etcher/commits/master this was a feature in v1.2.0 ??

Reply:
That's because the release branch itself was only merged after; but that definitely didn't end up in v1.2.0